content/en/blog/news/Delving_Deeper_Enriching_Microservices_with_Golang_and_Rust_with_CloudWeGo/index.md

```diff
@@ -0,0 +1,156 @@
+---
+date: 2024-02-21
+title: "Delving Deeper: Enriching Microservices with Golang and Rust with CloudWeGo"
+projects: ["CloudWeGo"]
+linkTitle: "Delving Deeper: Enriching Microservices with Golang and Rust with CloudWeGo"
+keywords: ["CloudWeGo", "middleware", "Kitex", "Netpoll", "Golang", "Volo", "rust", "microservice framework", "ByteDance Open Source", "ByteDance", "open source", "cloud native", "open source", "gRPC", "microservices", "rpc", "thrift"]
+description: "Delve into CloudWeGo's Kitex through practical examples, where high performance and extensibility redefine microservice excellence."
+author:  <a href="https://github.com/yy2so" target="_blank">Yacine Si Tayeb</a>
+---
+
+What if there existed an RPC framework that provided not only high performance and extensibility but also a robust suite of features and a thriving community support?
+
+CloudWeGo, a high-performance extensible Golang and Rust RPC framework originally developed and open-sourced by [ByteDance](https://opensource.bytedance.com), has caught my eye as it fits the bill perfectly.
+
+![Image](/img/blog/Delving_Deeper_Enriching_Microservices_with_Golang_and_Rust_with_CloudWeGo/1.jpeg)
+
+## CloudWeGo VS Other RPC Frameworks
+
+While [gRPC](https://grpc.io) and [Apache Thrift](https://thrift.apache.org) have served the microservice architecture well, [CloudWeGo](https://www.cloudwego.io)'s advanced features and performance metrics set it apart as a promising open source solution for the future.
+
+Built for the modern development landscape by embracing both [Golang](https://go.dev) and [Rust](https://www.rust-lang.org), CloudWeGo delivers advanced features and excellent performance metrics. As proof of its performance, benchmark tests have shown that [Kitex surpasses gRPC by 10 times](https://github.com/cloudwego/kitex-benchmark) in terms of QPS (Queries Per Second) and latency.
```

Comment:
Please update this based on original doc

Reply:
You mean the 10 times, right? The 10 times figure comes from the original doc. What's more appropriate to write? @GuangmingLuo